common/data/src/main/java/org/thingsboard/server/common/data/tenant/profile/DefaultTenantProfileConfiguration.java

```diff
@@ -57,6 +57,7 @@ public class DefaultTenantProfileConfiguration implements TenantProfileConfigura
     private long maxDPStorageDays;
     private int maxRuleNodeExecutionsPerMessage;
     private long maxEmails;
+    private boolean smsEnabled;
```

Comment:
Maybe just add Boolean instead of boolean and ```if smsEnabled == null``` it means that ```smsEnabled = true```? I just don't like the upgrade script :-( It seams very complex :-)